test/node/api-versions.js

```diff
@@ -58,9 +59,15 @@ test.test('must allow to specify version in application name', (test) => {
   test.end();
 });
 
+test.test('must set version to (1.0.0) if none was provided', (test) => {
```

Comment:
Why does this test have the same description as the one on line 221 in this file, do they test the same thing?

Reply:
I forgot to delete a test on line 221. It is deleted now.